doc/radosgw/multitenancy.rst

```diff
@@ -40,6 +40,8 @@ Create a user testx$tester to be accessed with Swift::
 
 Note that the subuser with explicit tenant had to be quoted in the shell.
 
+.. note:: Tenant names may only contain alphanumeric characters and underscores
+
 Accessing Buckets with Explicit Tenants
```

Comment:
@theanalyst Can you make it a single `Note` ? I think both sentences can be put under a single `..note::`, which is better to read and neat display.

```
Note that the subuser with explicit tenant had to be quoted in the shell.

.. note:: Tenant names may only contain alphanumeric characters and underscores
```